changelog.txt

```diff
@@ -1,5 +1,7 @@
 V.Next
 ----------
+- [MINOR] Enabling refresh_in feature (#1310)
+- [PATCH] Msal removeAccount shouldn't invoke broker's removeAccount (#1336)
```

Comment:
This seems to be coming from resolving a merge conflict I believe

Reply:
Sorry what is the issue here?

Reply:
The following shouldn't be here, right?

`- [PATCH] Msal removeAccount shouldn't invoke broker's removeAccount (#1336)`

Reply:
Why would should it not be?

Reply:
It is already released right? or have you done some work on it as part of your PR?

Reply:
Yes.  We released that change in 3.4.2, I think.